geoportal/c2cgeoportal_geoportal/views/entry.py

```diff
@@ -1249,6 +1269,7 @@ def loginuser(self):
     def loginchange(self):
```

Comment:
Not in PR scope, but loginchange => change_password, if we do not rename this method, a little docstring could be welcome.

Reply:
+1, done :-)